app/src/main/java/org/blitzortung/android/data/DataHandler.kt

```diff
@@ -139,7 +181,7 @@ class DataHandler @JvmOverloads constructor(private val wakeLock: PowerManager.W
                 val providerTypeString = sharedPreferences.get(PreferenceKey.DATA_SOURCE, DataProviderType.RPC.toString())
                 val providerType = DataProviderType.valueOf(providerTypeString.toUpperCase())
                 val dataProvider = dataProviderFactory.getDataProviderForType(providerType, sharedPreferences, agentSuffix)
-                this.dataProvider?.run { sharedPreferences.unregisterOnSharedPreferenceChangeListener(this@DataHandler.dataProvider) }
+                this.dataProvider?.run {sharedPreferences.unregisterOnSharedPreferenceChangeListener(this@DataHandler.dataProvider)}
```

Comment:
The spaces before and after the braces are due to the default formatting. Why should we remove them?